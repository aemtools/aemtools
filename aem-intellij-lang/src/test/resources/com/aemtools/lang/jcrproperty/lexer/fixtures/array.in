[val1, val-2, 3, jcr:property, with.dot, with space, with several spaces]
