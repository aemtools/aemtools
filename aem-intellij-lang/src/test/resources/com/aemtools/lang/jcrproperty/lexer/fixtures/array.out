[ ('[')
ARRAY_VALUE_TOKEN ('val1')
, (',')
ARRAY_VALUE_TOKEN (' val-2')
, (',')
ARRAY_VALUE_TOKEN (' 3')
, (',')
ARRAY_VALUE_TOKEN (' jcr:property')
, (',')
ARRAY_VALUE_TOKEN (' with.dot')
, (',')
ARRAY_VALUE_TOKEN (' with space')
, (',')
ARRAY_VALUE_TOKEN (' with several spaces')
] (']')
